xds/src/main/java/io/grpc/xds/orca/OrcaMetricReportingServerInterceptor.java

```diff
@@ -75,8 +84,11 @@ public <ReqT, RespT> Listener<ReqT> interceptCall(
         new SimpleForwardingServerCall<ReqT, RespT>(call) {
           @Override
           public void close(Status status, Metadata trailers) {
-            OrcaLoadReport report = fromInternalReport(
+            OrcaLoadReport.Builder reportBuilder = fromInternalReport(
+                InternalMetricRecorder.getMetricReport(metricRecorder));
```

Comment:
Note: `metricRecorder` could be `null` if a user writes (although unlikely) `OrcaMetricReportingServerInterceptor.create(null)`. Although non-nullability is implied unless marked otherwise, there is nothing preventing a user from doing this right now, should we add a check for null and throw in the constructor?

According to [go/java-practices/null](https://engdoc.corp.google.com/eng/doc/devguide/java/practices/null.md#tolerant), we should not be unnecessarily null tolerant. We could also add a null check here but it is more verbose:
```
OrcaLoadReport.Builder reportBuilder = metricRecorder != null ? fromInternalReport(
                InternalMetricRecorder.getMetricReport(metricRecorder)) :
                OrcaLoadReport.newBuilder();
```

Reply:
Done. In latest commit:
* `null` can be passed to the `create(MetricRecorder)` and the constructor. Added `@Nullable` annotation.
* `INSTANCE` to **not** use MetricRecorder, (e.g. `INSTANCE = new OrcaMetricReportingServerInterceptor(null)`)
* handle `null` in interceptor